pkg/backend/httpstate/stack.go

```diff
@@ -64,7 +64,7 @@ func (c cloudBackendReference) String() string {
 	// If the project names match, we can elide them.
 	if currentProject != nil && c.project == tokens.Name(currentProject.Name) {
 		// Elide owner too, if it is the default owner.
-		defaultOrg, err := pkgWorkspace.GetBackendConfigDefaultOrg(currentProject)
+		defaultOrg, err := placeholder.GetDefaultOrg(context.Background(), c.b, c.b.currentProject)
 		if err == nil && defaultOrg != "" {
 			// The default owner is the org
 			if c.owner == defaultOrg {
```

Comment:
How do DIY backends work wrt default orgs: Are they required to support organizations, individual organizations? If not, what happens when they run a pulumi command without specifying an org?

Reply:
DIY backends don't do anything with organizations.
Format for the stack identifier: you can do org/project/stack; if you're using that format, you use the constant ORGANIZATION for org. But any command that uses an org doesn't work with the DIY backend.

Is there a good testing strategy for DIY backend integration? -- follow up with justin